Terminal.Gui/Views/ScrollView.cs

```diff
@@ -385,6 +385,7 @@ public override void Redraw (Rect bounds)
 
 			// Fill in the bottom left corner
 			// BUGBUG: ScrollBarView should be responsible for this via contentBottomRightCorner
+			// this is only true if this is hosted by the ScrollBarView and it isn't
```

Comment:
Got it now. Thanks for clarifying. Please update the comment to read.

// Fill in the bottom left corner. Note we don't rely on ScrollBarView.contentBottomRightCorner here 
// because that only applies when `ScrollBarView` is hosted.